keyform/templates/keyform/home.html

```diff
@@ -1,6 +1,63 @@
 {% extends "keyform/base.html" %}
 {% load i18n %}
+{% load static %}
+{% block css %}
+    {{ block.super }}
+    <link rel='stylesheet' type='text/css' href="{% static 'keyform/css/home.css' %}"/>
+{% endblock %}
 
 {% block content %}
-    {% trans 'Welcome to Keyform!' %}
+    <h2 class="title">{% trans 'Keyform Requests' %}</h2>
+    <div class="object-list">
+        <div class="header row">
+            <div class="large-3 columns text-center">
```

Comment:
This display does not scale well to a smaller screen.  The foundation classes do what they're supposed to, but the columns are disassociated from each other in the smaller sizes.